### ISSUE TITLE
Poloniex Futures Api implementation.

### ISSUE THREAD

**Issue by firatsarlar** [jjxtra/ExchangeSharp#551]

**Comment by vslee** [comment]
Thanks for filing this feature request. If you send a pull request with the implementation, we'll be happy to merge it. If you don't know how, you can also post the updated code here.

**Comment by firatsarlar** [comment]
# @vslee 
Is there an implementation blueprint ?

**Comment by vslee** [comment]
Yes, you can take a look at other exchanges that support futures, such as [OKGroup](https://github.com/jjxtra/ExchangeSharp/blob/49735017048583ddd31850fe0b6a82de6040eaa3/src/ExchangeSharp/API/Exchanges/OKGroup/OKGroupCommon.cs), and implement something similar for [Poloniex](https://github.com/jjxtra/ExchangeSharp/blob/49735017048583ddd31850fe0b6a82de6040eaa3/src/ExchangeSharp/API/Exchanges/Poloniex/ExchangePoloniexAPI.cs).
